src/components/footer/Footer.css

```diff
@@ -0,0 +1,35 @@
+.footer-sub-container {
+    display: flex;
+    justify-content: space-between;
+    padding: 20px 0;
+    color: #ffffff;
+    font-size: 16px;
+    font-weight: 400;
+    line-height: 1.5;
+    border-bottom: 1px solid #e9ecef;
+    margin: 0 auto 100px auto;
+    width: 30%;
+}
+
+.footer-logo{
+    max-width: 100px;
+    margin: 30px 30px;
+
+}
+
+.footer-main-container {
```

Comment:
Als je nog eens goed naar de website kijkt, zie je dat de groepjes in de footer eigenlijk alle vijf even groot zijn (3x links, 1x locatie info en 1x formulier). Het is dus niet nodig om een linker en een rechter sub-groep te hebben. Dus je bent beter af met een container die deze vijf groepjes verdeeld met `justify-content: space-between`